tests/python/shared/utils/dump_objects.py

```diff
@@ -49,19 +50,36 @@ def clean_list_response(data: dict[str, Any]) -> dict[str, Any]:
     return data
 
 
+def _parse_asset_url(s: str) -> Tuple[str, str]:
+    asset_filename, url_path = s.lower().rsplit(":", maxsplit=1)
+    return asset_filename, url_path
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--assets-dir", type=Path, default=ASSETS_DIR)
+    parser.add_argument(
+        "--asset-path",
+        dest="asset_urls",
```

Comment:
These are inconsistent - either the argument should be `--asset-url`, or the variable should be `asset_paths`.

Reply:
Updated